src/pages/en/guides/images.md

```diff
@@ -91,42 +97,53 @@ import logo from '../images/logo.png';
 
 ### `public/`
 
-The [`public/` directory](/en/core-concepts/project-structure/#public) is for files and assets that do not need to be processed during Astro’s build process. Images stored here will be copied into the build folder untouched. These are not imported into your `.astro` file, and the image's `src` attribute is **relative to the public folder**.
+Your images stored in `public/` can be used by components (`.astro`, `.mdx` and other UI frameworks) and also Markdown files.
 
-```astro
----
-// src/pages/index.astro
 
+However, files in the `/public` directory are always served or copied as-is, with no processing. If you are using images outside of Markdown files, we recommend that local images are kept in `src/` when possible so that Astro can transform, optimize and bundle them.
+
+The `src` attribute is **relative to the public folder**. In Markdown, you can also use the `![]()` notation.
+
+```astro title="src/pages/index.astro"
+---
 // Access images in `public/images/`
 ---
 <img src="/images/logo.png" />
 ```
 
 ## Astro's Image Integration
 
-:::caution
-When you install the `@astrojs/image` integration, `.astro` files will no longer be able to use standard HTML `<img>` tags for images located in your project `src`. All local images must use the integration components instead.
-
-See the [image integration guide](/en/guides/integrations-guide/image/) for more notes about this new, experimental feature!
-:::
+Astro's official image integration provides two different Astro components for rendering optimized images, `<Image />` and `<Picture />`, as well as a basic image transformer. There is full support for both static sites and server-side rendering.
```

Comment:
Because Image and Picture sound like synonyms, I like differentiating them. But I'm not sure what I wrote here is any good.
If we don't document `getImage`, it might be confusing to mention here
```suggestion
Astro's official image integration provides two different Astro components for rendering optimized images, `<Image />` and `<Picture />`. `<Image/>` lets you optimize a single image, and `<Picture/>` lets you send a set of images for the most optimal one be chosen in the browser. There is full support for both static sites and server-side rendering.
```

Reply:
I agree with ditching the transformer, but I was keeping this simple here, and under each one I do say what they are for, and for Image, when they might prefer to use Picture. I *think* that's enough, and doesn't front-load the info?

Reply:
My thought was that we'd want to explain the difference upfront so they can understand why there are two without reading both sections - but I don't think it's totally necessary!